index.html

```diff
@@ -98,9 +125,33 @@ <h2>What have you achieved since Jan 2017?<br/>
             };
         })();
 
+        var salaryCount = (function () {
+            var monthlyBaseSalaryFromApr2016 = 49000 / 12;
+            var monthlyBaseSalaryFromApr2017 = 49500;
```

Comment:
Should this be divided by 12 as well, or am I misunderstanding? (quite possible!)

Reply:
It's a bit of a nomenclature one, but if you look at the `totalPay` function, it's effectively `monthlyBaseSalary * 12` but with the `* 12` factored out.

Reply:
Ah, I see now :+1:

Reply:
Should probably rename that to `annualSalaryFromApr2017` or something similar to avoid confusion.